Lib/fontbakery/profiles/universal.py

```diff
@@ -571,32 +573,32 @@ def com_google_fonts_check_required_tables(ttFont, config):
                        "BASE", "GPOS", "GSUB", "JSTF",
                        "gasp", "hdmx", "LTSH", "PCLT",
                        "VDMX", "vhea", "vmtx", "kern"]
-    # See https://github.com/googlefonts/fontbakery/issues/617
-    #
-    # We should collect the rationale behind the need for each of the
-    # required tables above. Perhaps split it into individual checks
-    # with the correspondent rationales for each subset of required tables.
-    #
-    # com.google.fonts/check/kern_table is a good example of a separate
-    # check for a specific table providing a detailed description of
-    # the rationale behind it.
-
-    optional_tables = [opt for opt in OPTIONAL_TABLES if opt in ttFont.keys()]
+
+    font_tables = ttFont.keys()
+
+    optional_tables = [opt for opt in OPTIONAL_TABLES if opt in font_tables]
     if optional_tables:
         yield INFO, \
               Message("optional-tables",
                       f"This font contains the following optional tables:\n\n"
                       f"{bullet_list(config, optional_tables)}")
 
-    if is_variable_font(ttFont):
-        # According to https://github.com/googlefonts/fontbakery/issues/1671
-        # STAT table is required on WebKit on MacOS 10.12 for variable fonts.
```

Comment:
Please do not erase knowledge ;-) Keep the comments, or reuse them somewhere else.

Reply:
Reinstated in 7a650fa7526a1884ff3462836289d94bad063c34